dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/analysis/PhenotypeAnalyzer.java

```diff
@@ -58,7 +58,7 @@
     // Here we eliminate duplicates and impose ordering (needed for reading the response items).
     val setIds = ImmutableSet.copyOf(entitySetIds).asList();
 
-    val multiResponse = donorRepository.performPhenotypeAnalysisMultiSearch(setIds);
+    val multiResponse = donorRepository.entitySetMultiSearch(setIds);
```

Comment:
findEntitySets would imply it is a general query using entity sets. This is a very specific multisearch with very specific aggregations.